style_guide/README.md

```diff
@@ -1,206 +1,307 @@
+   assign the appropriate version number.
+
+### Making hotfixes
+
+1. Create a new branch named `hotfix-<description>` off of the `main` branch, and commit any necessary changes following
+   the nominal `git add`/`git commit` cycle.
+2. Push the hotfix branch to the main `IMAP-Science-Operations-Center` `imap_processing` repo (i.e. `upstream`), and
+   open two separate pull requests: one that merges the hotfix branch into the `main` branch, and one that merges the
+   hotfix branch into the `dev` branch.
+3. For each of these pull requests, proceed with the nominal review & merge process described in steps (10) and (11) in
+   the previous section.
+4. Once the changes are merged into the `main` branch, create a
+   [new release](https://github.com/IMAP-Science-Operations-Center/imap_processing/releases) or a
+   [new tag](https://github.com/IMAP-Science-Operations-Center/imap_processing/tags) for the merge commit and assign the
+   appropriate [version number](#versioning).
+
+
+## Checklist for Contributors and Reviewers of Pull Requests
+
+The following is a guide to be used for contributors and reviewers of `imap_processing` pull requests. Note that this is
+only a guide; it should not be treated as a fully comprehensive, foolproof list that must be used in all situations, and
+parts of it are subjective.
+
+If the contributor/reviewer can answer "yes" to all the following questions, then conceivably the proposed changes are
+acceptable and the PR can be reviewed and merged.
+
+### Checklist for Contributors
+
+#### Pertaining to the code:
+
+- Does the code conform to this style guide?
+- Is any of the code functionality not already available via native or third-party python libraries?
+- Does the code execute successfully?
+    - Do all the tests pass in the existing test suite?
+    - Does the newly added functionality run without errors?
+- Is the code documented and commented sufficiently such that it is easy to read and follow?
+    - Are docstrings included for all new modules, classes, and functions?
+    - Are in-line comments included to provide necessary context?
+- Have all debugging/print statements been removed?
+- Does the code contain sufficient exception handling?
+- Does the code contain no deprecation warnings?
+- Does the code include all necessary unit tests?
+
+#### Pertaining to the pull request:
+
+- Is the PR excessively long and/or covers multiple issues? If so, consider breaking it up into multiple PRs.
+- Does the PR have a concise, descriptive title?
+- Does the PR link to and close the relevant issue?
+- Does the PR have a sufficient description as to make it clear what the reasons for the changes are?
+- Is the PR merging into `upstream/dev` from `<username>/<branchname>` (in most cases)?
+- Are you listed as an assignee to the PR?
+- Does the PR have proper labels?
+- Do all the automated checks pass?
+
+
+### Checklist for Reviewers
```

Comment:
This might make more sense in a PR template, rather than this file. I'm not sure PR reviewers will go back and read through the checklist every time